homeassistant/helpers/translation.py

```diff
@@ -170,7 +170,10 @@ async def _async_get_component_strings(
 
         for comp in components:
             domain, _, platform = comp.partition(".")
-            if not (integration := integrations.get(domain)):
+            if (
+                not (integration := integrations.get(domain))
+                or not integration.has_translations
+            ):
                 continue
 
             if platform and integration.is_built_in:
```

Comment:
We should drop the `if not has_files_to_load:` guard bellow so the title is always shimmed in 
`# Translations that miss "title" will get integration put in.`

That needs a test

Thats really an existing problem though.. its just now much more obvious